app/Models/VectorEmbedding.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Pgvector\Laravel\Vector;
+use Pgvector\Laravel\Distance;
+use Pgvector\Laravel\HasNeighbors;
+
+class VectorEmbedding extends Model
```

Comment:
The model class is not referenced anywhere in this PR. Is it needed in this codebase?